lib/project-view.coffee

```diff
@@ -10,6 +10,14 @@ module.exports = ProjectView =
       type: 'boolean'
       default: true
       description: 'Show the project path after project name in tree-view root.'
+    regexMatch:
+        type: 'string'
+        default: ""
+        description: 'Use a custom regex to transform the path'
```

Comment:
`Define a custom regex to find the parts in the project name that shall be replaced` sounds better.